bitmovin/resources/models/encodings/muxings/information/mp4_muxing_information.py

```diff
@@ -0,0 +1,92 @@
+from bitmovin.resources import Resource
+from bitmovin.errors import InvalidTypeError
+from bitmovin.resources.models.encodings.muxings.information.muxing_information_audio_track import \
+    MuxingInformationAudioTrack
+from bitmovin.resources.models.encodings.muxings.information.muxing_information_video_track import \
+    MuxingInformationVideoTrack
+
+from bitmovin.utils import Serializable
+
+
+class Mp4MuxingInformation(Resource, Serializable):
+
+    def __init__(self, mime_type=None, file_size=None, container_format=None, container_bitrate=None,
+                 duration=None, video_tracks=None, audio_tracks=None):
+        super().__init__()
+
+        self.mimeType = mime_type
+        self.fileSize = file_size
+        self.containerFormat = container_format
+        self.containerBitrate = container_bitrate
+        self.duration = duration
+        self._video_tracks = video_tracks
```

Comment:
please initialize self._video_tracks and self._audio_tracks with None and set self.video_tracks = video_tracks and the corresponding stuff for audio_tracks too so that the setters are called. that stuff shouldn't be directly set here